langchain4j-core/src/test/java/dev/langchain4j/store/embedding/TestUtils.java

```diff
@@ -0,0 +1,19 @@
+package dev.langchain4j.store.embedding;
+
+import org.awaitility.Awaitility;
+import org.awaitility.core.ThrowingRunnable;
+
+import java.time.Duration;
+
+public class TestUtils {
+    private TestUtils() {
+    }
+
+    public static void awaitUntilAsserted(ThrowingRunnable assertion) {
+        Awaitility.await()
+                .atMost(Duration.ofSeconds(60))
+                .pollDelay(Duration.ofSeconds(0))
+                .pollInterval(Duration.ofMillis(300))
+                .untilAsserted(assertion);
+    }
```

Comment:
Is there any reason not to use awaitility defaults mechanism?
```kotlin
Awaitility.setDefaultTimeout(..)
Awaitility.setDefaultPollInterval(..)
Awaitility.setDefaultPollDelay(..)
 ```
https://github.com/awaitility/awaitility/wiki/Usage#defaults

Reply:
This code was merged in as part of https://github.com/langchain4j/langchain4j/pull/1931 (after rebase, the commit that I mention in this PR description is no longer present here).

(I personally would not prefer this approach, as it removes code locality, which makes it difficult to know where the settings are set, or to be certain which awaits the defaults were intended to apply to.)